airflow/configuration.py

```diff
@@ -253,31 +239,61 @@ def run_command(command):
 # sync (default), eventlet, gevent
 worker_class = sync
 
+# Log files for the gunicorn webserver. '-' means log to stderr.
+access_logfile = -
+error_logfile = -
+
 # Expose the configuration file in the web server
-expose_config = true
+expose_config = False
```

Comment:
Oh thank god! It never made any sense to me why you'd want to view something like your fernet key in the UI